src/cursor/cur_hs.c

```diff
@@ -90,11 +94,11 @@ __wt_hs_cursor_next(WT_SESSION_IMPL *session, WT_CURSOR *cursor)
 }
 
 /*
- * __wt_hs_cursor_prev --
+ * __curhs_prev_int --
  *     Execute a prev operation on a history store cursor with the appropriate isolation level.
  */
-int
-__wt_hs_cursor_prev(WT_SESSION_IMPL *session, WT_CURSOR *cursor)
+static int
+__curhs_prev_int(WT_SESSION_IMPL *session, WT_CURSOR *cursor)
```

Comment:
I am not sure what is better option here - using `_int` or `_with_iso` suffix. @quchenhao your thoughts?

Reply:
I prefer int as in other places we all use this convention.

Reply:
All right then.

Reply:
We are operating on the file cursor here. May be __file_cursor_prev?

Reply:
hmm would that cause confusion, since __file_cursor_prev looks out of place in a hs cursor file?

Reply:
It is an internal function so the file cursor is in the context of hs cursor implementation.

Reply:
sounds good! changed